fiftyone/operators/__init__.py

```
@@ -12,6 +12,7 @@
     list_operators,
     operator_exists,
 )
+from .evaluation_metric import EvaluationMetricConfig, EvaluationMetric
```

Ruff (0.8.2)

`.evaluation_metric.EvaluationMetricConfig` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)